crates/evm/core/src/era_revm/db.rs

```diff
@@ -55,9 +55,26 @@ impl<DB: Database + Send> RevmDatabaseForEra<DB>
 where
     <DB as revm::Database>::Error: Debug,
 {
-    /// Returns the current block number and timestamp from the database.
+    /// Create a new instance of [RevmDatabaseForEra] caching the current l2 block.
+    pub fn new(db: Arc<Mutex<Box<DB>>>) -> Self {
+        let db_inner = db.clone();
+        let current_block = {
+            let mut db = db_inner.lock().unwrap();
```

Comment:
Can we start handling `unwraps` or should we wait to have proper `Errors` defined?